rosidl_generator_py/resource/_msg.py.template

```diff
@@ -103,7 +103,9 @@ class @(spec.base_type.type)(metaclass=Metaclass):
 @[      end if]@
 @[      if field.type.type == 'byte']@
         self.@(field.name) = kwargs.get('@(field.name)', b'0')
-@[      elif field.type.type == 'char']@
+@[      elif field.type.type in [
+            'char',
+            'string']]@
```

Comment:
Why would a Python string field be initialized with a `\0`? Shouldn't it be an empty string?